docs/source/install.rst

```diff
@@ -0,0 +1,35 @@
+Installation
+============
+
+To use Dask-CUDA on your system, you will need the following:
+
+- At least one CUDA-compatible GPU
+- NVIDIA CUDA Toolkit
+- *Anything else?*
```

Comment:
Technically, Dask-CUDA won't interface directly with the CUDA Toolkit, so it's not exactly a dependency. However, it will be a dependency for other libraries that will be used for CUDA computation, like CuPy or Numba. IOW, the compute libraries will need to satisfy the CUDA Toolkit dependency, and not Dask-CUDA.

Reply:
Good to know - is there anything in particular that is required to use Dask-CUDA? Right now I'd imagine:

- CUDA-compliant GPU
- Some installation of CUDA

On that topic, I think to address the fact that the install instructions used to direct users to install cuDF + the toolkit, maybe I'll add a section at the bottom of this page directing to the general RAPIDS quickstart guide.

Reply:
It's difficult to say what is particularly required, we intentionally try to make it as unrestricted as possible. It's definitely true that CUDA GPUs and a CUDA installation are required, but this is actually necessary by the downstream libraries that will make use of the GPU itself, as Dask-CUDA will not interact directly with the GPU. However, Numba is a dependency and [Numba CUDA support depends on CUDA Toolkit](https://numba.pydata.org/numba-doc/latest/cuda/overview.html#software). Perhaps we should then indeed say CUDA Toolkit is a requirement, but that isn't available in PIP so the user needs to get it elsewhere. We're fine with providing PIP packages, but we shouldn't encourage it, so maybe we should intentionally limit documentation on it.

Reply:
Thanks for the explanation! In that case, CUDA toolkit should probably go in the list of main requirements, and then I think I'll put CUDA and the compliant GPUs as requirements for the optional RAPIDS libraries.

One question - when Conda installing Dask-CUDA, is the CUDA toolkit version specified (like `cudatoolkit=11.2`) to install the toolkit alongside Dask-CUDA, or to make sure that its dependencies are using versions that support the system installation of the toolkit? If it's the latter, do we know how Pip handles the CUDA toolkit version? If not, then that makes a pretty good case to recommend Conda install and removing the Pip instructions altogether.